firmware/chipcon_usb.c

```diff
@@ -1168,6 +1169,10 @@ void processOUTEP5(void)
                 appReturn(2, ptr);
                 break;
 
+            case CMD_DEVICE_SERIAL_NUMBER:
```

Comment:
make this point to the usb serial number (stored just after the bootloader and before the beginning of firmware) and we're gtg.